dependencies.yaml

```diff
@@ -73,7 +73,8 @@ libcurl:
 # of libocpp and would otherwise be overwritten by the version used there
 libevse-security:
   git: https://github.com/EVerest/libevse-security.git
-  git_tag: v0.9.7
+  # FIXME update to "v0.9.8" when https://github.com/EVerest/libevse-security/pull/119 is merged
```

Comment:
FIXME: The tag string needs to be updated once https://github.com/EVerest/libevse-security/pull/119 is merged and the release is tagged there.